src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -38,6 +38,12 @@
     </EmbeddedResource>
   </ItemGroup>
   
+  <Target Name="WorkaroundNetStandard" AfterTargets="ResolvePackageAssets">
```

Comment:
Make sure you have an issue tracking the removal of this workaround.

Reply:
Thanks! Tracking issue is raised: https://github.com/NuGet/Home/issues/8364

Reply:
Thanks! Raised https://github.com/NuGet/Home/issues/8364 accordingly.